pom.xml

```diff
@@ -518,9 +517,9 @@
                 <version>${mail.version}</version>
             </dependency>
             <dependency>
-                <groupId>javax.portlet</groupId>
-                <artifactId>portlet-api</artifactId>
-                <version>${portlet-api.version}</version>
+                <groupId>org.apache.portals</groupId>
+                <artifactId>portlet-api_2.1.0_spec</artifactId>
+                <version>1.0</version>
```

Comment:
Strange choice, IMHO, on the part of the folks releasing this new API jar.  Since the meaning is stripped from the <version> number, I support the notion of taking it out of the <properties> section;  putting 1.0 there would be confusing.